birthdays-project/birthday/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class BirthdayConfig(AppConfig):
```

Comment:
This only makes sense if you add the config to the INSTALLED_APPS instead of the app name. So in `base.py`, replace `BirthdayApp` with `BirthdayApp.apps.BirthdayConfig`